### PR TITLE
[13.0][ADD] delivery_preference_glue_stock_picking_group

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

File: delivery_preference_glue_stock_picking_group/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+class StockMove(models.Model):
+
+    _inherit = "stock.move"
+
+    def release_available_to_promise(self):
+        """After release reset to customer prefered carrier the backorder."""
+        pickings = (
+            self.filtered(self._filter_recompute_preferred_carrier)
```
Comment (simahawk) [inline delivery_preference_glue_stock_picking_group/models/stock_move.py]: couldn't you put your condition in AND with `_filter_recompute_preferred_carrier`?

File: delivery_preference_glue_stock_picking_group/readme/ROADMAP.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+This is a glue module, so the question is.
+
+Is it necessary or there is a bug some where than cause this ?
+
+Maybe in the grouping done by the module `stock_picking_group_by_partner_by_carrier`
```
Comment (simahawk) [inline delivery_preference_glue_stock_picking_group/readme/ROADMAP.rst:5]: eh, good question :smile:

**Review by simahawk — COMMENTED** [review]
I'm not sure we need it :thinking: 
I'd discuss w/ Joël and JE as I have to solve a similar issue in another context.

File: delivery_preference_glue_stock_picking_group/models/stock_move.py (hunk 1 above)
Comment (TDu) [reply delivery_preference_glue_stock_picking_group/models/stock_move.py]: I changed that.

**Comment by TDu** [comment]
@simahawk This is already used as a local addon. It is only a move to the OCA, plus the tests.

**Review by guewen — APPROVED** [review]

**Pull request merged**